src/harness/unittests/jsDocParsing.ts

```diff
@@ -288,5 +288,19 @@ namespace ts {
   */`);
             });
         });
+        describe("getFirstToken", () => {
+            it("gets jsdoc", () => {
+                const first = ts.createSourceFile("foo.ts", "/** comment */var a = true;", ts.ScriptTarget.ES5, /*setParentNodes*/ true);
+                assert.isDefined(first);
+                assert.equal(first.kind, 263);

```

Comment:
Oops. I grabbed `263` straight from the debugger. Nice catch. It looks like this issue needs some more attention (to fix `getNextToken` as well) so I'll correct this in the followup PR.